### PR TITLE
Update "Turkey" to "Türkiye"

### LINKED ISSUE #439: Update TR display
Turkey's country name has been updated to Türkiye. 

https://www.un.org/en/about-us/member-states/turkiye

As such, both ISO and IANA have update the display long and short names as of 2022-07-11. 
https://www.iso.org/obp/ui/#iso:code:3166:TR
```
%%
Type: region
Subtag: TR
Description: Türkiye
Description: Turkey
Added: 2005-10-16
%%
```

We've already had clients inform of using the new display name everywhere as needed. Will add `Legacy territory name: "Turkey"` to the notes field on update, as well as updating Display Name.

### PR DESCRIPTION
**Pull request by SteveLLamb**
Closes #439, updates as per notes in issue, country name has been update in UN, as as such ISO and IANA. 

Also had to update schema to allow for `À-ÿ` within territory name.

### CODE REVIEW

**Review by palemieux — CHANGES_REQUESTED** [review]
Increment the schema version number?

**Comment by SteveLLamb** [comment]
> Increment the schema version number?

Yes, let me update the site side first to use the "latest", rather than being hard coded, so that we can be seamless when merging this PR.

File: src/main/schemas/territories.schema.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       },
       "dcncTerritory": {
         "type": "string",
-        "pattern": "^[a-zA-Z ']+$"
+        "pattern": "^[a-zA-ZÀ-ÿ ']+$"
```
Comment (palemieux) [inline src/main/schemas/territories.schema.json:18]: What about allowing all Unicode characters but for control characters?
```suggestion
        "pattern": "^[^\x00-\x1f\x7f\x80-\x97]+$"
```

**Review by palemieux — CHANGES_REQUESTED** [review]

File: src/main/schemas/territories.schema.json (hunk 1 above)
Comment (SteveLLamb) [reply src/main/schemas/territories.schema.json]: Updated via [43d5ff3](https://github.com/ISDCF/registries/pull/440/commits/43d5ff3da268f5191b787abd4c0fe93623899c64)

**Review by palemieux — APPROVED** [review]

**Comment by SteveLLamb** [comment]
> Yes, let me update the site side first to use the "latest", rather than being hard coded, so that we can be seamless when merging this PR.

Not going to do this after all. We still want schema version validation.

**Pull request merged**